src/main/java/edu/kis/vh/stacks/IStack.java

```diff
@@ -2,7 +2,8 @@
 
 public interface IStack {
 	
-	final int STACK_EMPTY = -1;
+	final int STACK_EMPTY = 0;
+	// ulatwilo, lecz musialem poprawic rowniez testy
```

Comment:
Niestety, ale testy poprawia się tylko jak są błędne. W tym przypadku to kod jest błędny.